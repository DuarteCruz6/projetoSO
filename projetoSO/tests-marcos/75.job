WRITE [(b2,3)(d3,14)(i0,26)]
WRITE [(n7,13)]
WRITE [(l2,45)(v9,41)(r1,34)(p7,8)(m3,31)]
READ [d3]
WRITE [(q8,91)(l4,44)]
WRITE [(e6,82)(z3,13)(v4,73)(v6,63)(l9,100)]
WRITE [(i2,90)(y4,13)(q6,60)]
DELETE [l2,e6]
WAIT 1000
WRITE [(v1,83)(f4,93)(e3,5)]
WRITE [(a4,5)(w9,29)]
WRITE [(t3,23)]
WRITE [(j6,44)(a0,64)(q5,50)(h0,99)(t6,51)]
WRITE [(n4,52)(u9,98)(m2,64)(h5,8)]
WRITE [(l1,4)(g0,69)(o4,44)]
WRITE [(p1,44)(g9,72)(y0,13)(v5,57)(b6,88)]
READ [g9]
WRITE [(u8,77)]
WRITE [(q1,98)(c3,68)(o2,71)(r0,95)(d6,5)]
WRITE [(d2,57)(h6,19)(i8,43)(s7,92)]
WRITE [(x3,72)(x5,58)(l0,26)]
WRITE [(w5,24)(k0,96)(u0,44)]
DELETE [q8,f4,n7,b2]
WAIT 1000
WRITE [(w8,59)(w6,32)]
WRITE [(v0,92)]
WRITE [(t5,6)(g7,79)(m1,68)]
DELETE [p1,v5,s7,q5,p7]
WRITE [(c6,31)(n1,82)(b4,65)(x9,83)]
WAIT 3000
DELETE [e3]
READ [t3]
WRITE [(c9,79)]
WRITE [(i9,31)(x2,23)(j1,2)(h3,40)]
DELETE [l9]
WAIT 2000
WRITE [(g1,12)(s5,5)(m0,27)(r4,53)(x4,22)]
WRITE [(t9,67)(a1,90)(q7,43)]
WRITE [(n0,28)(y3,20)(v7,25)(x0,9)]
WAIT 3000
SHOW
WRITE [(i4,56)(b2,49)(j9,87)]
WRITE [(k9,39)(b8,3)]
WRITE [(l3,28)(t7,20)(z5,29)(e9,77)(t4,10)]
WAIT 1000
READ [o4]
WRITE [(e1,11)(e1,27)(n5,81)(o5,75)]
WRITE [(j2,86)(g2,60)(o0,33)]
BACKUP
WRITE [(a9,5)(z4,31)(i1,21)(o8,6)]
READ [k0]
WRITE [(w2,33)(e8,53)(i6,32)]
READ [m2]
SHOW
WRITE [(y9,15)(r8,42)]
WRITE [(k3,4)]
WRITE [(l8,30)(g5,7)]
WRITE [(c8,75)(z6,73)(h7,91)(o3,100)(y6,48)]
WRITE [(p4,56)(u7,82)]
BACKUP
WRITE [(z9,56)(z7,20)(f4,18)(n7,72)(d7,59)]
WRITE [(r3,71)(s0,69)(k6,98)]
BACKUP
WRITE [(c2,42)(d8,70)(z2,80)(o1,17)(m6,60)]
WRITE [(f7,72)]
READ [g7]
READ [q7]
WRITE [(d4,15)(h8,87)(z8,87)(a6,88)(s4,19)]
READ [j9]
READ [d6]
WRITE [(e0,51)]
DELETE [x3,c6,i6,l1,d7]
