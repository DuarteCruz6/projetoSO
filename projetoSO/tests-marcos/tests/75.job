WRITE [(q9,66)(z7,14)(i5,77)(e8,49)]
WRITE [(g1,35)(a9,54)(a0,100)]
WRITE [(w8,77)(m8,79)(l6,81)(o5,40)]
WRITE [(g5,98)(t2,87)]
WRITE [(c4,48)(z6,29)(k4,72)(e6,88)(v7,10)]
WRITE [(f9,0)(f4,42)]
WAIT 2000
READ [f9]
WRITE [(n3,2)(e9,91)(s5,43)]
READ [e9]
WRITE [(k7,36)(p8,70)(d4,16)]
DELETE [w8,g1]
DELETE [s5,e9]
DELETE [f4,v7,e8,t2,l6]
READ [g5]
READ [o5]
WRITE [(w7,89)(m3,55)]
DELETE [z6]
READ [w7]
WRITE [(m1,20)(b9,31)(a1,7)(y8,93)]
DELETE [y8]
READ [k4]
WRITE [(j4,40)(g7,6)]
SHOW
WRITE [(y1,6)(l7,7)(c7,83)(r3,74)(i3,70)]
WRITE [(z1,57)(x0,93)(n7,16)(h0,27)(f5,97)]
WRITE [(l5,79)(d3,20)(w8,84)(t4,67)]
WRITE [(u2,83)(p6,89)(v2,51)(w2,61)(j0,30)]
WRITE [(r2,81)(t7,53)(s7,83)(v4,42)]
WRITE [(u9,5)(r7,45)(r5,21)(u8,58)(l0,37)]
READ [b9]
READ [u2]
WRITE [(k2,99)]
WRITE [(f1,7)(e5,87)(b3,16)(b4,69)]
WRITE [(h5,2)(j7,12)(d5,71)(c9,84)]
