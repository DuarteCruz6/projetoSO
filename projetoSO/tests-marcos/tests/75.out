[(f9,0)]
[(e9,91)]
[(g5,98)]
[(o5,40)]
[(w7,89)]
[(k4,72)]
(a1, 7)
(a0, 100)
(a9, 54)
(b9, 31)
(c4, 48)
(d4, 16)
(e6, 88)
(f9, 0)
(g7, 6)
(g5, 98)
(i5, 77)
(j4, 40)
(k7, 36)
(k4, 72)
(m1, 20)
(m3, 55)
(m8, 79)
(n3, 2)
(o5, 40)
(p8, 70)
(q9, 66)
(w7, 89)
(z7, 14)
[(b9,31)]
[(u2,83)]
